backend/globaleaks/handlers/admin/notification.py

```diff
@@ -157,3 +159,33 @@ def put(self):
 
         self.set_status(202)
         self.finish(response)
+
+class EmailNotifInstance(BaseHandler):
+    """
+    Send Test Email Notifications to the admin that clicked the button.
+    Understand that this handler blocks its thread until both the db query
+    and the SMTP round trip return.
+    """
+
+    @BaseHandler.transport_security_check('admin')
+    @BaseHandler.authenticated('admin')
+    @inlineCallbacks
+    def post(self):
+      """
+      Errors: InvalidAuthentication, ForbiddenOperation, TODOstmpError
+      """
+      user = yield get_user_settings(self.current_user.user_id, 
+                               GLSettings.memory_copy.default_language)
```

Comment:
The alignment we have used so far is with the [G]Lsettigs aligned with [s]elf.current but i do not know if it's standards or not